### PR TITLE
Aurelou/ethereum fork

### PR DESCRIPTION
**Pull request by 0xAurelou**
Hello Mangrove team,

This small PR aims to support Ethereum Fork.

What have been changed : 

- Create ``Ethereum.sol`` in the ``test/lib/forks/`` folder
- Add ``Ethereum`` variable in the ``foundry.toml`` file
- Modify ``.env.example`` 

Do not hesitate to make feedbacks 😉

### CODE REVIEW

File: test/lib/forks/Ethereum.sol [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier:	AGPL-3.0
+pragma solidity ^0.8.10;
+
+import {GenericFork} from "./Generic.sol";
+
+contract MumbaiFork is GenericFork {
```
Comment (peterMangrove) [inline test/lib/forks/Ethereum.sol]: Should probably be named EthereumFork, and not MumbaiFork

**Comment by adhusson** [comment]
Thanks a lot for this PR and congrats on being our first external contributor :) 🤝🎉 Welcome!

File: test/lib/forks/Ethereum.sol (hunk 1 above)
Comment (0xAurelou) [reply test/lib/forks/Ethereum.sol]: Thanks Peter forgot to update it.
Acknowledge and fixed

**Review by espendk — APPROVED** [review]
lgtm. Thank you @0xAurelou 🙏

**Comment by 0xAurelou** [comment]
Hello Guys,

I just signed and squashed all the commit,

If you want to It is ready to merge 🚀

**Review by espendk — APPROVED** [review]

**Comment by 0xAurelou** [comment]
Hello guys,

Thanks for the review and the kind words,
Happy to help, as Usual it's always a pleasure to work with you.

Happy to be your first external contributor.

🚀

**Pull request merged**